pkgs/top-level/all-packages.nix

```
@@ -11689,6 +11689,8 @@ with pkgs;
     nodejs = nodejs_latest;
   };
 
+  wasmedge = callPackage ../development/tools/wasmedge { };
```

Use `llvmPackages` as the argument in the package, so updates to the LLVM version suggested by upstream won't require changing all of the `12`s in the package names.

```suggestion
  wasmedge = callPackage ../development/tools/wasmedge {
    llvmPackages = llvmPackages_12;
  };
```

---

Thanks. Done.